changes/ce/fix-14001.en.md

```diff
@@ -0,0 +1 @@
+Fixed a race condition where a resource (connector/action/source/authentication/authorization) could report having a connected, healthy channel where in actuality it was broken, after a moment of disconnection.
```

Comment:
could you describe what errors or logs may be observed from the result of this race ?

Reply:
Since it can affect any of these resources, logs and symptoms will vary wildly.

From the originating issue, in the case of HTTP action, it'll print `action_not_found`:

```
{"time":"2024-10-14T00:06:40.337936+00:00",
  "level":"error","msg":"action_not_found",
  "rule_trigger_ts":[1728864400337],"action_id":"action:http:action-7669bcb7:connector:http:connector-7b556bfa",
  "client_ids":{"8988228066609179058":true}, 
  "rule_ids":{"rule-16c5b355":true},"connector":"connector:http:connector-7b556bfa","pid":"<0.6443.0>","line":454}
```

I don't think there's a way to list all possible logs.